mvp-spec/signatures.md

```diff
@@ -0,0 +1,62 @@
+# Signatures
+
+A quick introduction on how data (id, preferences) and messages (operator API, RTB) are signed and how these signatures can be verified.
```

Comment:
id = "Pseudonymous-identifiers".

messages = "Operator requests and Transmissions"

Note that Transmissions aren't necessary in RTB

Reply:
reword pushed, please close the conversation if you agree.